h/static/scripts/polyfills.js

```diff
@@ -31,3 +31,23 @@ require('keyboardevent-key-polyfill');
 // Fetch API
 // https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API
 require('whatwg-fetch');
+
+// Quick polyfill to normalize the naming of matches function
+// from https://developer.mozilla.org/en/docs/Web/API/Element/matches
+if (!Element.prototype.matches) {
+  Element.prototype.matches =
+    Element.prototype.matchesSelector ||
+    Element.prototype.mozMatchesSelector ||
+    Element.prototype.msMatchesSelector ||
+    Element.prototype.oMatchesSelector ||
+    Element.prototype.webkitMatchesSelector ||
+    function(s) {
+      const matches = (this.document || this.ownerDocument).querySelectorAll(s);
```

Comment:
Hmm, this seems a little bit inefficient as it searches the whole document rather than just the relevant subtree. But hey, we can fix that it if becomes a problem.